extern/storage-sealing/sealing.go

```diff
@@ -224,15 +224,23 @@ func (m *Sealing) Remove(ctx context.Context, sid abi.SectorNumber) error {
 
 // Caller should NOT hold m.unsealedInfoMap.lk
 func (m *Sealing) StartPacking(sectorID abi.SectorNumber) error {
+	// locking here ensures that when the SectorStartPacking event is sent, the sector won't be picked up anywhere else
+	m.unsealedInfoMap.lk.Lock()
+	defer m.unsealedInfoMap.lk.Unlock()
+
+	// cannot send SectorStartPacking to sectors that have already been packed, otherwise it will cause the state machine to exit
```

Comment:
This...shouldn't have been a problem, since the FSM's `planOne` should safely skip an unexpected `SectorStartPacking` event, since it's an `Ignorable`.

(Don't see anything wrong with adding this check, though)